src/index.ts

```diff
@@ -263,7 +261,7 @@ export const staticPlugin = async <Prefix extends string = '/prefix'>(
 
             if (indexHTML && pathName.endsWith('/index.html'))
                 app.get(
-                    join(prefix, pathName.replace('/index.html', '')),
+                    pathName.replace('/index.html', ''),
```

Comment:
This is a fix for `always static with assets on an absolute path` once the prefix was fixed in test. The previous version of tests was why the wrong test passed and the commented test didn't